config/samples/sfptpd/overlays/example-sf0/sfptpd.cfg

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: MIT
+# SPDX-FileCopyrightText: (c) Copyright 2023 Advanced Micro Devices, Inc.
+
+[general]
+sync_module ptp ptp1
+
+# Log to stdout/stderr. The recommended way to deploy would be only logging
+# the message log, to stderr, and placing an adapter program in the pod to
+# process RT JSON stats and make them available to a time series database.
+message_log stderr
+stats_log stdout
+
+[ptp1]
+ptp_domain 0
+interface sf0
```

Comment:
This I think needs a ptp_mode master line.

Reply:
No it doesn't `example-sf0` is for customers to use/tailor for their environment which will have an external third party grandmaster. Your change converts this into a working alternative to the "link-local" example which therefore needs changing and renaming.